src/microbe_stage/AvailableUpgrade.cs

```diff
@@ -7,7 +7,15 @@
 /// </summary>
 public class AvailableUpgrade : IRegistryType
 {
+    private LoadedSceneWithModelInfo loadedSceneData;
+
 #pragma warning disable 169,649 // Used through reflection
+    /// <summary>
+    ///   A path to a scene to override organelle's display scene. If empty won't have a display model.
```

Comment:
> If empty won't have a display model.

Isn't this wrong or very annoying? Because upgrades that don't specify visual changes should just use the normal organelle graphics.